### PR TITLE
Cypress/fe 1774 select bdd specs

### PR DESCRIPTION
**Pull request by DlgSHi**
Added select default / multiple bdd specs

### CODE REVIEW

File: cypress/features/regression/select.feature [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+    Then label is set to "<label>"
+    Examples:
+      | label                    |
+      | Sample text              |
+      | 1234567890               |
+      | áéíóú¿¡üñ                |
+      | !@#$%^*()_+-=~[];:.,?{}  |
+      | ÄÖÜßäöüß                 |
+      | <>                       |
+
```
Comment (marcinstachowicz) [inline cypress/features/regression/select.feature]: reduntant newline

File: cypress/features/regression/select.feature [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+    Then Select is enabled
+
+  @positive
+  Scenario: Select is readOnly
+    When I check readOnly checkbox
+    Then Select is readOnly
+
+  @positive
+  Scenario: Select is not readOnly
+    Given I check readOnly checkbox
```
Comment (marcinstachowicz) [inline cypress/features/regression/select.feature]: When is missing

File: cypress/features/regression/selectMultiple.feature [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+    Then Select is enabled
+
+  @positive
+  Scenario: Select is readOnly
+    When I check readOnly checkbox
+    Then Select is readOnly
+
+  @positive
+  Scenario: Select is not readOnly
+    Given I check readOnly checkbox
```
Comment (marcinstachowicz) [inline cypress/features/regression/selectMultiple.feature]: When is missing

File: cypress/features/regression/selectMultiple.feature [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+    Then label is set to "<label>"
+    Examples:
+      | label                    |
+      | Sample text              |
+      | 1234567890               |
+      | áéíóú¿¡üñ                |
+      | !@#$%^*()_+-=~[];:.,?{}  |
+      | ÄÖÜßäöüß                 |
+      | <>                       |
+
```
Comment (marcinstachowicz) [inline cypress/features/regression/selectMultiple.feature]: redundant new line

File: cypress/support/step-definitions/select-steps.js [hunk 5]
```diff
@@ -0,0 +1,71 @@
+import { select, selectInput, selectPill } from '../../locators/select';
+
+Then('Select typeAhead is disabled', () => {
+  select()
+    .should('have.attr', 'placeholder', 'Please Select...');
```
Comment (marcinstachowicz) [inline cypress/support/step-definitions/select-steps.js]: Here you break line of select.should() but below not. Please unify and break only if line is too long

File: cypress/features/regression/select.feature (hunk 1 above)
Comment (DlgSHi) [reply cypress/features/regression/select.feature]: fixed

File: cypress/features/regression/select.feature (hunk 2 above)
Comment (DlgSHi) [reply cypress/features/regression/select.feature]: fixed

File: cypress/features/regression/selectMultiple.feature (hunk 3 above)
Comment (DlgSHi) [reply cypress/features/regression/selectMultiple.feature]: fixed

File: cypress/features/regression/selectMultiple.feature (hunk 4 above)
Comment (DlgSHi) [reply cypress/features/regression/selectMultiple.feature]: fixed

File: cypress/support/step-definitions/select-steps.js (hunk 5 above)
Comment (DlgSHi) [reply cypress/support/step-definitions/select-steps.js]: unified

**Pull request merged**